Tabloid/Controllers/TagController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tabloid.Models;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
This controller should be auth protected.

Reply:
I believe I got the auth for it working.